doc(release) Release notes oss 24.10 avr.26 (#5243)
centreon/centreon-documentation: versioned_docs/version-24.10/releases/centreon-os.mdx

| Centreon AWIE | [24.10.5 (February 25, 2026)](#centreon-awie-24105)<br/>[24.10.4 (January 26, 2026)](#centreon-awie-24104)<br/>[24.10.3 (December 31, 2025)](#centreon-awie-24103)<br/>[24.10.2 (December 18, 2025)](#centreon-awie-24102)<br/>[24.10.1 (March 10, 2025)](#centreon-awie-24101) |
| Centreon High Availability | [24.10.2 (December 18, 2025)](#centreon-high-availability-24102)<br/>[24.10.1 (August 6, 2025)](#centreon-high-availability-24101)<br/>[24.10.0 (October 31, 2024)](#centreon-high-availability-24100) |

</details>

## April 30, 2026

### Centreon Web 24.10.24

<details open>
  <summary>Enhancements</summary>

- Added examples for the usage of AND / OR in API documentation.
- [Performance] Improved SQL query construction for better performance in Resource Status.
- [Performance] Removed deprecated row-counting method and removed an unnecessary SQL join to optimize queries in Resource Status.

</details>

<details>
  <summary>Bug fixes</summary>

> - Fixed an issue on host edition/creation API.

Reviewer: ```suggestion
- Fixed an issue where calling the host edition/creation API resulted in a "You are not authorized to access this resource" error.
```

> - Fixed an issue where `otel_server` block was added to `otl_server.json` when only 'Connection initiated by poller' was active.

Reviewer: ```suggestion
- Fixed an issue where an `otel_server` block was added to `otl_server.json` when only 'Connection initiated by poller' was active.
```

> - [ACL] Fixed issue with ressources access and host group.

Reviewer: ```suggestion
- [ACL] Fixed an issue where ACL rights were not applied correctly to host groups, resulting in a "You are not authorized to access this resource" error.
```

> - [Administration] Fixed an issue where additional configuration wasn't updated after deleting user.

Reviewer: ```suggestion
- [Administration] Fixed an issue where additional configurations weren't updated after deleting a user.
```

> - [Administration] Fixed an issue where unlocking a locked contact fails with SQL syntax error.

Reviewer: ```suggestion
- [Administration] Fixed an issue where unlocking a locked contact failed with an SQL syntax error.
```

> - [Authentication] Fixed SAML authentication failure. Please manually verify your SAML comparison settings if you previously used a non-default configuration.

Reviewer: ```suggestion
- [Authentication] Fixed SAML authentication failure.
   > This update may affect custom SAML authentication configurations. If you changed any SAML settings from their default values, manually verify that your configuration still works as expected.
```

> - [Configuration] Fixed an issue on additional configuration when changing vCenter name.

Reviewer: ```suggestion
- [Configuration] Fixed an issue where changing the name of a vCenter instance in an additional configuration was causing an error.
```

> - [Dashboards] Duplicate entries are now removed from the metric selection.

Reviewer: ```suggestion
- [Dashboards] The metric selection list no longer displays duplicate entries.
```

> - [Event Logs] Fixed an issue in filter dropdowns where the service list failed to update based on the selected host.

Reviewer: ```suggestion
- [Event Logs] Fixed an issue with filters where the service list failed to update after a host was selected.
```

- [Graph] Legend names now display correctly instead of showing the "Default" placeholder.
- [Remote server] Fixed an issue where tasks were not correctly imported.

> - [Trapsnmp] Fixed credentials exposure in centreontrapd.sdb.

Reviewer: ```suggestion
- [Trapsnmp] Fixed the SQLite database setup for SNMPTrapD to only include custom macros for resources monitored by the poller.
```

> - [WEB] Fixed console error from css MIME type.

Reviewer: ```suggestion
- [WEB] Fixed incorrect MIME type for CSS files, causing an error in the browser console.
```

- [Administration] Fixed an issue where additional configurations weren't updated after deleting a user.
- [Administration] Fixed an issue where unlocking a locked contact failed with an SQL syntax error.
- [Authentication] Fixed SAML authentication failure.
   > This update may affect custom SAML authentication configurations. If you changed any SAML settings from their default values, manually verify that your configuration still works as expected.
- [Configuration] Fixed an issue where changing the name of a vCenter instance in an additional configuration was causing an error.
- [Dashboards] The metric selection list no longer displays duplicate entries.
- [Event Logs] Fixed an issue with filters where the service list failed to update after a host was selected.
- [Graph] Legend names now display correctly instead of showing the "Default" placeholder.
- [Remote server] Fixed an issue where tasks were not correctly imported.
- [Trapsnmp] Fixed the SQLite database setup for SNMPTrapD to only include custom macros for resources monitored by the poller.

> - [WEB] Fixed incorrect MIME type for CSS files, causing an error in the browser console.

Reviewer: ```suggestion
- [WEB] Fixed incorrect MIME type for CSS files that caused an error in the browser console.
```
